execute_pipeline.py

```diff
@@ -145,6 +145,8 @@ def _load_config_spec(config_spec, config_sections, repl_vars):
         all_config_data = yaml.load(config_file)
     for section in config_sections:
         data.update(all_config_data[section])
+    if language in all_config_data:
```

Comment:
But conversely, say we add a special section, and the user wants to specify that, but also they don't want to have to include the language too because that would be redundant with the --language arg. 

I guess I would argue that if someone needs an exact list of sections and wants to exclude --language, maybe we should make special syntax for that when it actually comes up as a use case.